.github/workflows/cws-btfhub-sync.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Install python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.8.14'
+          python-version: '3.9.5'
```

Comment:
Maybe a quick question why 3.9.5 and not 3.9.18 (the latest at this time) ?

Reply:
Because it's the version in the buildimages used by the CI (see `Motivation` section).